source/style/style.css

```diff
@@ -29,13 +29,25 @@ body {
   scroll-behavior: smooth;
 }
 
+/* Main Display */
+#add-book,
+#contact-section {
+  display: none;
+}
+
 .hero-container {
   display: flex;
   flex-direction: column;
   width: 50%;
   margin: 0 auto;
 }
 
+.hero-container h2 {
+  margin-bottom: 2rem;
+  font-size: 2rem;
+  text-align: center;
+}
+
 .title {
   text-align: center;
   font-size: 2rem;
```

Comment:
- [ ] Great job so far, everything works fine, please try to resemble the wireframe. for example you don't have the line separating the navigation links.

<img width="293" alt="Screen Shot 2022-10-26 at 12 32 07 PM" src="https://user-images.githubusercontent.com/59416136/197991063-45067bfc-6d8e-4d6b-9f97-a23be48232d2.png">